docs/solutions/Hadoop/EL7/Cloudera/N-Node/env-vars

```diff
@@ -0,0 +1,30 @@
+### Authentication details
+export TF_VAR_tenancy_ocid=""
+export TF_VAR_user_ocid=""
+export TF_VAR_fingerprint=""
+export TF_VAR_private_key_path="/home/opc/.ssh/oci_api_key.pem"
+
+### Region
+export TF_VAR_region="us-phoenix-1"
+
+### Compartment
+export TF_VAR_compartment_ocid=""
+
+### Public/private keys used on the instance
+export TF_VAR_ssh_public_key=$(cat /home/opc/.ssh/id_rsa.pub)
+export TF_VAR_ssh_private_key=$(cat /home/opc/.ssh/id_rsa)
+
+## AD
+export TF_VAR_AD="2"
+
+## OS Image OCID
+export TF_VAR_image_ocid="ocid1.image.oc1.phx.aaaaaaaazs4djfpanpbh7zwqjqe4enku3juqo6dntkj66u2vjx4p2k2oj5cq"
```

Comment:
See the image OCID pattern used in other examples. (Essentially, use a map to allow for different regions.)